alembic/util/exc.py

```diff
@@ -1,6 +1,16 @@
+from typing import TYPE_CHECKING
```

Comment:
we use future annotations, so we can avoid the " form the typing below

Reply:
I'm not sure to understand your point here.
Removing solely the quotes makes the symbol not found, and we don't need the import at runtime so it's generally a good practice to keep it behind a TYPE_CHECKING control

Reply:
That's where future annotation comes in

See for example https://github.com/sqlalchemy/alembic/blob/980bd91afbaa5fa60c3aa2c7661114f1bfa4a100/alembic/command.py#L3

Reply:
ok, today I learned :D 

I've just pushed an update for both your comments